petclinic-frontend/src/pages/Users/AllUsers.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from 'react';
+import './AllUsers.css';
+import { NavBar } from '@/layouts/AppNavBar.tsx';
+import axios from 'axios';
+
+const AllUsers: React.FC = (): JSX.Element => {
+  interface UserResponseModel {
```

Comment:
As shown in the image under, the roles are being sent to the front-end. As an admin, seeing the roles of the users is very important and needed. They can be added in the table or could be used to seperate the tables into who has which role, I'll leave that to your discretion but as long as they're being shown as other pull requests after you depend on having the role.
![image](https://github.com/user-attachments/assets/23ec21c3-9225-4446-b76f-0a2bf5e64830)

Reply:
Yes i agree with you I will add that